File: Seamana 01/09-ejercicio.cpp
#include <iostream>
using namespace std;
float a, b;
int main() {
    cout <<"ingrese el primer numero: ";
    cin >> a;
    cout <<"ingrese el segundo numero: ";
    cin >> b;
    if (a > b) {
        cout << a <<" es mayor que " << b << endl;
    }
    else if (a<b) {
        cout << a <<" es menor que " << b << endl;
    }
    else {
        cout << a <<" es igual a " << b << endl;
    }
    return 0;
}

File: Seamana 01/PYTHON/09-ejercicio.py
print("ingrese el primer numero: ")
a = float(input())
print("ingrese el segundo numero: ")
b = float(input())

if a > b:
    print(a, "es mayor que", b)
elif a < b:
    print(a, "es menor que", b)
else:
    print(a, "es igual a", b)